src/EnergyPlus/CurveManager.cc

```diff
@@ -2088,11 +2087,19 @@ namespace CurveManager {
                         ErrorsFound = true;
                     } else if (pointsSpecified) {
                         // normalizeGridValues normalizes to 1.0 at the reference values. We must redivide by passing in the 1.0/normalizationDivisor as the scalar here.
-                        btwxtManager.normalizeGridValues(gridIndex, PerfCurve(CurveNum).GridValueIndex, normalizeTarget, 1.0/normalizationDivisor);
+                        normalizationDivisor = btwxtManager.normalizeGridValues(gridIndex, PerfCurve(CurveNum).GridValueIndex, normalizeTarget, normalizationDivisor)*normalizationDivisor;
                     }
+                }
 
-              }
+                // Perform Divisor Normalization for methods: NM_DIVISOR_ONLY || NM_AUTO_WITH_DIVISOR
+                PerfCurve(CurveNum).NormalizationValue = normalizationDivisor;
 
+                if ((normalizeMethod == NM_DIVISOR_ONLY && PerfCurve(CurveNum).CurveMaxPresent) || (normalizeMethod == NM_AUTO_WITH_DIVISOR && PerfCurve(CurveNum).CurveMaxPresent)){
+                    PerfCurve(CurveNum).CurveMax = PerfCurve(CurveNum).CurveMax / normalizationDivisor;
+                }
+                if ((normalizeMethod == NM_DIVISOR_ONLY && PerfCurve(CurveNum).CurveMinPresent) || (normalizeMethod == NM_AUTO_WITH_DIVISOR && PerfCurve(CurveNum).CurveMinPresent)){
```

Comment:
Same here

Reply:
Hi Julien. I agree that this looks like a better evaulation method. I have changed the code to match.